peachjam/views/courts.py

```diff
@@ -67,14 +64,26 @@ def get_context_data(self, **kwargs):
             }
         )
 
-        registries = list(
+        registries = [
             {
-                registry
-                for registry in self.form.filter_queryset(
-                    self.get_base_queryset(), exclude="registries"
-                ).values_list("registry__name", flat=True)
-                if registry
+                "name": registry["name"],
+                "code": registry["code"],
+                "url": reverse(
```

Comment:
If you define `get_absolute_url()` on the Registry class, then you can simply use registry objects here, rather than copying the name and code into a new dictionary just so that you can include a url with the information. So the registry object knows how to build its own URL.